docs/includes/nugetsolver-tool.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Include
+description: Suggest NuGetSolver experimental tool.
+author: ErickYondon
+ms.author: eryondon
+ms.date: 1/16/2024
+ms.topic: include
+---
+
+Use NuGetSolver, our experimental tool for solving dependency conflicts, to help you find solutions to problems like this automatically!
```

Comment:
This should explicitly say that NuGet Solver is not officially supported. Also, instead of saying our, it should say "from Microsoft DevLabs".

Reply:
How about this one?
https://review.learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1107?branch=pr-en-us-3210
![image](https://github.com/NuGet/docs.microsoft.com-nuget/assets/8766776/62dd17ef-5678-4883-827d-0c3bd7ff3bab)